Makefile

```diff
@@ -85,8 +85,8 @@ else
   endif
 endif
 
-# DEFAULT means these targets will always be executed
-.DEFAULT: all 
+# Force the rebuild of targets - CI relies on this.
```

Comment:
Perhaps skip the "CI relies on this", it does rely on it now but we can easily change it so it doesn't.
We do not have to set this practice in stone. Maybe just "# Force the rebuild of targets".

Reply:
Sure - one sec.

Reply:
Also, what do you think about letting Windows have its .exe-suffixes?
If the codebase grows, it would be time consuming to always force rebuilds, but here we seem to be OK with it. And then the renaming thing is a non-issue, so perhaps we don't have to bother?

Reply:
I'd like to see if we can figure out the Cuda Linker output file issue - I haven't given up hope yet. Will report back on the investigation either in this PR or the CI issue we have.